portfolio/src/main/webapp/scripts/controllers/comments_controller.js

```diff
@@ -77,11 +76,15 @@ const addNewCommentToList = newComment => {
 const url = 'https://socket-dot-scarletnguyen-step-2020.uk.r.appspot.com/';
 const socket = io.connect(url);
 socket.on("comment-updates", newComment => {
-  if (newComment.message !== undefined && newComment.message === "deleted") {
-    requestComments('GET', null);
-  } else {
-    addNewCommentToList(newComment);
-  }
+  if (newComment !== undefined) {
+    if (newComment.deleted !== undefined) {
```

Comment:
The semantic meaning of the deleted field is unclear to me here.  Why would userId = newComment.deleted?  Make this more clear.